src/components/index.js

```diff
@@ -2,3 +2,5 @@ export { Navbar } from "./Navbar/Navbar";
 export { CategoriesCard } from "./CategoriesCard/CategoriesCard";
```

Comment:
in every folder name the js file "index.js" so that it will auto-import that file only from the folder

Reply:
You are in that file only.